core/src/main/java/com/yahoo/oak/BlockMemoryAllocator.java

```diff
@@ -28,8 +28,8 @@
     // Returns the memory allocation of this OakMap (this Allocator)
     long allocated();
 
-    // Read the buffer from an allocation object that its parameters are set: blockID, offset and length
-    void readByteBuffer(Slice s);
+    // Attaches the slice with its base address
+    void getSlicebaseAddress(Slice s);
```

Comment:
coding convention requires name to be `getSliceBaseAddress()`

Reply:
Isn't it then `setSliceAddress()`? Is it setting the base address? Or the direct, unique address of this Slice?

Reply:
here we are setting the base address

Reply:
lets discuss